components/layout/UserMenu.vue

```diff
@@ -22,7 +22,7 @@
         <small>{{ user.role }}</small>
       </hc-navbar-item>
       <hr class="navbar-divider">
-      <hc-navbar-item class="nav-item" to="/profile" @click.native="close()">
+      <hc-navbar-item class="nav-item" @click="userSlug()" @click.native="close()">
```

Comment:
You should avoid navigating by method as the accessibility is not given when the link href is not propagated correctly. 

You could change the method to a computed property and pass it like ```:to="userProfile"```

You can also check the accessibility of an element (links etc.) by using the icon at the bottom left of the page in development.

Reply:
I modified my approach @appinteractive, I originally tried to use a computed property, but couldn't get it working, and I couldn't find any other instances in the code base that use that approach. 
I did address your underlying concern though and no longer use a method to navigate. Let me know what you think. :smile:

Reply:
Perfect! Thank you 🙏😁

Reply:
Thank you!! The code has been improved for sure!! :smiley: